src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -5,8 +5,7 @@
 using System.Windows.Controls;
 using Dynamo.Controls;
 using Dynamo.Extensions;
-using Dynamo.Logging;
-using Dynamo.Selection;
+=using Dynamo.Selection;
```

Comment:
@mjkkirschner  There is an extra "=" here causing the build to fail

Reply:
thanks!